src/backend/aspen/workflows/pangolin/save.py

```diff
@@ -48,6 +48,7 @@ def cli(pangolin_fh: io.TextIOBase, pangolin_last_updated: datetime):
                 "lineage": row["lineage"],
                 "probability": get_probability(row),
                 "version": row["version"],
+                "full_output": {k: v for k, v in row.items() if v and k != "taxon"},
```

Comment:
Filter out empty fields and any fields whose key is "taxon" since that's just the sample id.